filters/all.js

```diff
@@ -13,3 +13,28 @@ function pathResolve(pathName, basePath = '/') {
   return path.resolve(basePath, pathName);
 }
 filter.pathResolve = pathResolve;
+
+/*
+ * returns comma separated string of all operationIds of a given channel
+*/
+function getOperationIds(channel) {
+  const noOperationIdError = 'This template requires operationId to be set in every operation.'
+  let list = '';
+  let parseOperationId = (channel, opName) => {
+    const id = opName === 'subscribe' ? channel.subscribe().id() : channel.publish().id();
+    if (!id) throw new Error(noOperationIdError);
+
+    return `, ${id}`;
+  }
+
+  if (channel.hasSubscribe()) {
+    list += parseOperationId(channel, 'subscribe');
+  }
+
+  if (channel.hasPublish()) {
+    list += parseOperationId(channel, 'publish');
+  }
+
+  return list.substring(1);
+}
```

Comment:
You can use here `join` function:

```suggestion
const parseOperationId = (channel, opName) => {
  const id = opName === 'subscribe' ? channel.subscribe().id() : channel.publish().id();
  if (!id) throw new Error('This template requires operationId to be set in every operation.');
  return id;
}

function getOperationIds(channel) {
  const list = [];
  if (channel.hasSubscribe()) {
    list.push(parseOperationId(channel, 'subscribe'));
  }
  if (channel.hasPublish()) {
    list.push(parseOperationId(channel, 'publish'));
  }
  return list.filter(Boolean).join(', ');
}
```